### ISSUE TITLE
Set all operators to active/inactive

### ISSUE THREAD

**Issue by JuMooly** [GoodEffort/Arknights-Planner#24]
Would it be possible to add an "Activate all" and "Deactivate all" button? It's a bit cumbersome to manually deactivate 50 operators when I want to check what just a couple of them would cost real quick, not to mention having to reactivate them all again.

**Comment by GoodEffort** [comment]
Yeah, that's an easy addition, I can add that when I redesign that section!
